### PR TITLE
Release @metamask/spdx-compare v1.1.0

### PR DESCRIPTION
**Pull request by legobeat**
This is the first release as `@metamask/spdx-compare`.

Changes since fork from upstream `spdx-compare@1.0.0`:

- Remove dependency `array-find-index` (#2) 
- docs: fix usage example for Node.js >= v10 (#1)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "spdx-compare",
+  "name": "@metamask/spdx-compare",
   "description": "compare SPDX license expressions",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (mcmire) [inline package.json:4]: Seems like all of the changes are fixes. Should this be 1.0.1?

Alternatively, conventionally we've bumped major when renaming. Should this be 2.0.0?

Comment (legobeat) [reply package.json:4]: I figured it as an improvement (apart from docs and test its more of an optimization). But makes sense to see it as a fix as well!

Seeing name-only change as breaking is a new stance to me and I can see the case of it even if it seems more principal then practical. Makes sense.

I'm OK with either of these - got a strong preference for either I can reopen this as?

Comment (mcmire) [reply package.json:4]: Yeah, let's go with 2.0.0 just so it's one more thing that stands out with this release.

Comment (legobeat) [reply package.json:4]: #5

**Comment by legobeat** [comment]
- Superseded by #5

**Pull request closed without merging**